React/tic-tac/src/app/screens/game/components/square/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import style from '../square/style.scss';
```

Comment:
The `style.scss` file is within the same folder, you can reference it from there. 🤓